modules/persistent-storage-hostpath-about.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-hostpath.adoc
+
+[id="persistent-storage-hostpath-about_{context}"]
+= Overview
+
+{product-title} supports hostPath mounting for development and testing on a single-node cluster.
+
+In a production cluster, you would not use hostPath. Instead, a cluster administrator would provision a network resource, such as a GCE Persistent Disk volume, an NFS share, or an Amazon EBS volume. Network resources support the use of storage classes to set up dynamic provisioning.
+
```

Comment:
Maybe we can remove `an NFS share` from here. NFS share should not be a recommended storage in the product cluster, and in OCP311, we don't support dynamic provisioning for NFS share.